rllib/policy/sample_batch.py

```diff
@@ -1631,7 +1635,22 @@ def _concat_values(*values, time_major=None) -> TensorType:
         time_major: Whether to concatenate along the first axis
             (time_major=False) or the second axis (time_major=True).
     """
-    return np.concatenate(list(values), axis=1 if time_major else 0)
+    if torch and isinstance(values[0], torch.Tensor):
```

Comment:
Can you explain why we need these changes (add comment)?

Reply:
Also, let's use `torch.is_tensor()`.

Reply:
done.